src/main/java/appeng/tile/storage/TileIOPort.java

```diff
@@ -107,9 +109,6 @@ public TileIOPort()
 
 		final Block ioPortBlock = AEApi.instance().definitions().blocks().iOPort().maybeBlock().get();
```

Comment:
Just as a quick notice. While technically correct, the tile itself is only present when the block is present itself. So it already has an implicit `isPresent()` check. But sometimes I feel like we should add a quick comment as someone might not immediately realise the transitive relation. Not now, just something to think about.